design/2082-bsl-delete-associated-resources_design.md

```diff
@@ -0,0 +1,42 @@
+# Delete Backup and Restic Repo Resources when BSL is Deleted
+
+## Abstract
+
+Issue #2082 requested that with the command `velero backup-location delete <bsl name>` (implemented in Velero 1.6 with #3073), the following will be deleted:
+
+- associated backups
```

Comment:
Let's be very clear here that these are the Kubernetes resources stored in the API server.